openduck-py/openduck_py/routers/voice.py

```diff
@@ -126,18 +126,79 @@ def __call__(self, audio_data):
 
 
 class ResponseAgent:
-    def __init__(self, session_id: str, output_sample_rate=24_000):
+    def __init__(
+        self,
+        session_id: str,
+        input_audio_format: Literal["float32", "int32", "int16"] = "float32",
```

Comment:
Not for this PR but it would be helpful to use headers to specify data formats/types as done here: https://github.com/toverainc/willow-inference-server/blob/main/main.py#L1341-L1345